crates/agglayer-clock/src/block.rs

```diff
@@ -372,9 +382,16 @@ impl PubSubConnect for WsConnectWithRetries {
                 // progress when the client is disconnected
                 fail::fail_point!("block_clock::PubSubConnect::try_reconnect::add_delay");
 
-                Ok(self.0.try_reconnect().await?)
+                let handle = self
+                    .connection
+                    .try_reconnect()
+                    .await
+                    .inspect_err(|err| debug!(?err, "Failed to reconnect to the L1 node"))?;
```

Comment:
Nit: this should probably be an info or a warning. We would probably like to know if there are issues with the L1 node communication from the standard logs.